### PR TITLE
mariadb100 10.2.28

### PR DESCRIPTION
**Pull request by kuahyeow**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-versions/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
- [x] Does your submission adhere to Versions "Acceptable Formulae" [guidelines](https://github.com/Homebrew/homebrew-versions/blob/master/README.md#acceptable-formulae)?

-----

### CODE REVIEW

File: mariadb100.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Mariadb100 < Formula
   end
 
   option :universal
-  option "with-tests", "Keep test when installing"
+  option "with-test", "Keep test when installing"
```
Comment (MikeMcQuaid) [inline mariadb100.rb]: Please use a `deprecated_option` to rename this.

**Comment by MikeMcQuaid** [comment]
Some audit failures: https://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/1755/version=sierra/testReport/junit/brew-test-bot/sierra/audit_homebrew_versions_mariadb100/

**Comment by MikeMcQuaid** [comment]
Fails to build on 10.11 too: https://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/1755/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/install_homebrew_versions_mariadb100/

File: mariadb100.rb (hunk 1 above)
Comment (kuahyeow) [inline mariadb100.rb]: Have addressed this, thanks. And the audit failures too

**Comment by kuahyeow** [comment]
I'm not sure why it failed to build on 10.11 on the bot. I successfully built the formula locally on 10.11.6. Will look closer

**Comment by kuahyeow** [comment]
Built it again locally, was successful. I'm stumped for now

Here's the error from the build bot : 

```
cd /tmp/mariadb100-20161208-87976-1o4y8jg/mariadb-10.0.28/dbug && ./factorial 1 2 3 4 5 > output1.r
[ 28%] Linking CXX executable tests
cd /tmp/mariadb100-20161208-87976-1o4y8jg/mariadb-10.0.28/dbug && /usr/local/Cellar/cmake/3.7.1/bin/cmake -E cmake_link_script CMakeFiles/tests.dir/link.txt --verbose=1
dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
  Referenced from: /private/tmp/mariadb100-20161208-87976-1o4y8jg/mariadb-10.0.28/dbug/./factorial
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _clock_gettime
  Referenced from: /private/tmp/mariadb100-20161208-87976-1o4y8jg/mariadb-10.0.28/dbug/./factorial
  Expected in: /usr/lib/libSystem.B.dylib

/bin/sh: line 1: 94337 Trace/BPT trap: 5       ./factorial 1 2 3 4 5 > output1.r

```

**Comment by MikeMcQuaid** [comment]
@kuahyeow Try `system "cmake", ".", *std_cmake_args, *args`

**Comment by kuahyeow** [comment]
Oh yay, that worked. Thanks @MikeMcQuaid

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**